rfcs/0029-allow-script-multiple-matches-on-identical-code/0029-allow-script-multiple-matches-on-identical-code.md

```diff
@@ -1 +1,93 @@
-In review, see <https://github.com/nervosnetwork/rfcs/pull/222>
+---
+Number: "0029"
+Category: Consensus (Hard Fork)
+Status: Draft
+Author: Ian Yang
+Organization: Nervos Foundation
+Created: 2021-02-03
+---
+
+# Allow Multiple Cell Dep Matches When There Is No Ambiguity
+
+## Abstract
+
+This document proposes a transaction verification consensus change to allow multiple cell dep matches on type script hash when all the matches are resolved to the same script code.
+
+## Motivation
```

Comment:
I would recommend a re-org in this section:

* First we talked about the fact that there are 2 ways to reference a cell
* Now we introduce the problem: multiple matches for a single type script hash would be an error in current CKB(possibly an example to illustrate this), then we can argue the legitimacy of allowing multiple matches